deployments/relayer/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Okay, we can remove this yaml file(Ingress) for relayer! 🙇🏼‍♂️

Reply:
I removed the ingress file